modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectDefinitionVulcanBatchEngineTaskItemDelegateTest.java

```diff
@@ -182,6 +182,7 @@ private ObjectDefinition _createObjectDefinition(String name) {
 					RandomTestUtil.randomString());
 				id = RandomTestUtil.randomLong();
 				label = Collections.singletonMap("en_US", "O" + sanitizedName);
+				modifiable = true;
```

Comment:
Can you change this logic in order to have the value true just when the object definition is a custom object?